src/locales/fr-FR/stream.ftl

```diff
@@ -250,6 +250,15 @@ comments-jumpToComment-GoToReply = Aller à la réponse
 comments-permalink-copyLink = Copier le lien
 comments-permalink-linkCopied = Lien copié
 
+comments-refreshComments-closeButton =
+  .aria-label = Fermer
+comments-refreshComments-refreshButton =
+  .aria-label = Rafraîchir les commentaires
+comments-refreshQuestions-refreshButton =
+  .aria-label = Rafraîchir les questions
+  comments-refreshReviews-refreshButton =
```

Comment:
can you update to remove the space at the start of this line?

Reply:
Hello @tessalt yes I am sorry, I have just removed it